google-oauth-client/src/main/java/com/google/api/client/auth/oauth/OAuthHmacSigner.java

```diff
@@ -32,14 +34,34 @@
 @Beta
 public final class OAuthHmacSigner implements OAuthSigner {
```

Comment:
class doc comment needs to be updated

Reply:
this has been reverted back in favor of a new class